Userland/Libraries/LibIPC/Connection.cpp

```diff
@@ -84,6 +81,12 @@ void ConnectionBase::shutdown()
     die();
 }
 
+void ConnectionBase::shutdown_with_error(Error const& error)
+{
+    dbgln("IPC::ConnectionBase ({:p}) had error ({}), disconnecting.", this, error);
```

Comment:
```suggestion
    dbgln("IPC::ConnectionBase ({:p}) had an error ({}), disconnecting.", this, error);
```

Reply:
Copy pasted from other code; fixed in both places.